src/api/app/views/webui/package/_show_actions.html.haml

```diff
@@ -2,7 +2,7 @@
   %li.nav-item.action-report-bug
     = render partial: 'webui/package/show_actions/bugzilla_owner', locals: { url: package.report_bug_or_bugzilla_url }
   - if User.session
-    - if current_rev && is_working
+    - if current_rev && is_working && is_branchable
       = render partial: 'webui/package/show_actions/branch_package', locals: { package: package, project: project,
                                                                                              revision: revision }
       = render partial: 'webui/package/show_actions/submit_package', locals: { package: package, project: project,
```

Comment:
```suggestion
    - if current_rev && is_working && is_branchable
      = render partial: 'webui/package/show_actions/branch_package', locals: { package: package, project: project,
                                                                                             revision: revision }
    - if current_rev && is_working
      = render partial: 'webui/package/show_actions/submit_package', locals: { package: package, project: project,
```
Submitting a package is inside this check. We want to prevent from branching a package, but... do we also want to prevent from submitting a package?

Reply:
no, indeed not. thanks!